deploy/infra-cluster-service-account.yaml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
remove the leading 2 spaces for this file

Reply:
done